### PR TITLE
Add arcade publishing step

### PR DESCRIPTION
**Pull request by JoeRobich**
Test build showing publishing stages will run https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3909640&view=results

### CODE REVIEW

File: eng/Signing.props [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<Project>
+  <PropertyGroup>
+    <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileFullPath)</MSBuildAllProjects>
+  </PropertyGroup>
```
Comment (sharwell) [inline eng/Signing.props:4]: 📝 This is no longer necessary (MSBuild provides the feature automatically, with lower overhead)

Comment (JoeRobich) [reply eng/Signing.props:4]: It complained with there was no signing.props file. I will remove this property group.

Comment (sharwell) [reply eng/Signing.props:4]: Right, I was specifically referring to this property group.

Comment (JoeRobich) [reply eng/Signing.props:4]:
```suggestion
```

**Comment by JoeRobich** [comment]
After upgrading Arcade symbol Validate Build assets stage completes - https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=3931729&view=results

**Pull request merged**